src/Test.UnitTests.Sarif/Writers/SarifLoggerTests.cs

```diff
@@ -511,7 +511,7 @@ public void SarifLogger_ScrapesFilesFromResult()
             {
                 using (var sarifLogger = new SarifLogger(textWriter,
                                                          analysisTargets: null,
-                                                         dataToInsert: OptionallyEmittedData.Hashes,
+                                                         dataToInsert: OptionallyEmittedData.None,
```

Comment:
>None [](http://example.com/codeflow?start=93&length=4)

Interesting bug! Because we didn't have sufficient handling for the `dataToInsert` property, we missed a case where this data was requested but not available. The fix now prompts request for missing data. Since that's not the purpose of this test, I just dropped the request for hashes data.